### PR TITLE
hmmclassify

### PR DESCRIPTION
**Pull request by erikrikarddaniel**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metatdenovo/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/metatdenovo _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

I reimplemented the hmmsearch/hmmrank function.

### CODE REVIEW

File: workflows/metatdenovo.nf [hunk 1]
```diff
@@ -252,7 +250,7 @@ workflow METATDENOVO {
         PROKKA_CAT(ch_assembly_contigs)
         ch_versions = ch_versions.mix(PROKKA_CAT.out.versions)
         ch_gff      = PROKKA_CAT.out.gff.map { it[1] }
-        ch_hmm_aa   = PROKKA_CAT.out.faa
+        //ch_hmm_aa   = PROKKA_CAT.out.faa
```
Comment (danilodileo) [inline workflows/metatdenovo.nf]: Do we need to remove this comment? there are other comments inside this script. was it a test?

Comment (erikrikarddaniel) [reply workflows/metatdenovo.nf]: I forgot to mention this when we spoke. I couldn't see  a reason why there are two channels for amino acid output, but I didn't dare to just remove so I commented them out. I'll run some tests with the other orf callers to see that all works, then remove the commented lines.

File: workflows/metatdenovo.nf [hunk 2]
```diff
@@ -100,19 +97,22 @@ include { FASTQC_TRIMGALORE } from '../subworkflows/local/fastqc_trimgalore'
 include { DIGINORM } from '../subworkflows/local/diginorm'
 
 //
+// Is this really a good heading? I assume it's taken from the example above (INPUT_CHECK), but does one really need to know what's in the subworkflow? Or should we collect all subworkflows under the possible three headings: only nf-core, only local, mix?
```
Comment (danilodileo) [inline workflows/metatdenovo.nf:100]: Just saw this comment. we could probably just collect the subworkflows under those three headings. I'm not sure about the guidelines but it should be fine

**Review by danilodileo — APPROVED** [review]
LGTM

**Pull request merged**